src/PrestaShopBundle/Entity/Tab.php

```diff
@@ -97,7 +97,7 @@ class Tab
     /**
      * @var bool
      *
-     * @ORM\Column(name="hide_host_mode", type="boolean")
+     * @ORM\Column(name="hide_host_mode", type="boolean", options={"default":"1"})
      */
     private $hideHostMode;
```

Comment:
@PululuK are you sure it's related to `hide_host_mode` not `enabled`? 🤔

Reply:
I see a lot of advertisements on TV that say: 'alcohol abuse is dangerous' ... I now understand why.
Done... Thanks @kpodemski !